src/api/color.rs

```diff
@@ -41,33 +41,34 @@ pub use v_frame::pixel::ChromaSampling;
 /// Supported Color Primaries
 ///
 /// As defined by “Color primaries” section of ISO/IEC 23091-4/ITU-T H.273
+#[wasm_bindgen]
```

Comment:
wasm_bindgen should not require this, can you poke upstream and ask them to fix that in the macro?

Reply:
Will do

Reply:
https://github.com/rustwasm/wasm-bindgen/issues/2273